helmctlr/helm.go

```diff
@@ -31,7 +31,7 @@ var defaultNS = "default"
 // Controller is a crwatcher.ResourceController that works with Helm to deploy
 // helm charts into K8s providing a CustomResource as value data to the charts
 type Controller struct {
-	ChartDir    string         // path to dir where the Helm chart is located
+	ChartPath   string         // path to dir where the Helm chart is located
```

Comment:
Comment needs updating to reflect that this could be the path to a chart archive and not just a dir.